src/test/unit/stick_commands_unittest.cc

```diff
@@ -0,0 +1,329 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
```

Comment:
New licence file please see default licence in repo root.

Reply:
@zvonler https://github.com/betaflight/betaflight/blob/master/DEFAULT_LICENSE.md